controllers/taloscontrolplane_controller.go

```diff
@@ -307,7 +307,9 @@ func (r *TalosControlPlaneReconciler) Reconcile(ctx context.Context, req ctrl.Re
 			if err := r.bootstrapCluster(ctx, cluster, ownedMachines); err != nil {
 				conditions.MarkFalse(tcp, controlplanev1.MachinesBootstrapped, controlplanev1.WaitingForTalosBootReason, clusterv1.ConditionSeverityInfo, err.Error())
 
-				return ctrl.Result{}, err
+				logger.Info("Bootstrap failed, retrying in 5 seconds", "error", err)
+
+				return ctrl.Result{RequeueAfter: time.Second * 5}, nil
```

Comment:
is 5 seconds too aggressive here? should we do something like 20 seconds? which was DefaultRequeueTimeout constant somewhere

Reply:
yeah, I wanted to have it aggressive :) Though I'm ok to change it if you think that it's overkill.

Reply:
`DefaultRequeueTimeout` is defined only in Sidero

Reply:
yep, we could do some when we want to retry some operation